ccdproc/tests/test_image_collection.py

```diff
@@ -217,6 +217,17 @@ def test_generator_data(self, triage_setup):
         for img in collection.data():
             assert isinstance(img, np.ndarray)
 
+    def test_generator_ccddata(self, triage_setup):
+        from ccdproc import CCDData
```

Comment:
I think you can import this at the top of the file. It's likely someone does need in at some point for another test.